py3-platformdirs.yaml

```diff
@@ -0,0 +1,44 @@
+# Generated from https://pypi.org/project/platformdirs/
+package:
+  name: py3-platformdirs
+  version: 3.10.0
+  epoch: 0
+  description: A small Python package for determining appropriate platform-specific dirs, e.g. a "user data dir".
+  copyright:
+    - license: "MIT"
+  dependencies:
+    runtime:
+      - py3-typing-extensions
+      - python3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - python3
+      - py3-setuptools
+      - py3-build
+      - py3-installer
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/platformdirs/platformdirs
+      tag: ${{package.version}}
+      expected-commit: 7035de4344d2ae4e55774c4bdec838f0a59a03fd
+
+  - name: Python Build
+    runs: |
+      python -m build
+      python3 -m installer -d "${{targets.destdir}}" dist/*.whl
+      find ${{targets.destdir}} -name "*.pyc" -exec rm -rf '{}' +
```

Comment:
(not for this PR) I see this line somewhat often, should this be its own pipeline (like a Python strip or something)?